wagtail/core/models.py

```diff
@@ -670,13 +671,47 @@ def save(self, clean=True, **kwargs):
                 self.url_path
             )
 
+        if log_action is not None:
+            # The default for log_action is False. i.e. don't log unless specifically instructed
+            # Page creation is a special case that we want logged by default, but allow skipping it
+            # explicitly by passing log_action=None
+            if is_new:
```

Comment:
> Don't we need to disable this for page copy?

@kaedroho, we can't do this without a context manager (which we agreed would be overkill). *sigh*

1. we want log page creation generally
2. it probably is a good idea to disable this for page copy, however the copy happens via `Page._copy()` which calls `_save_copy_instance()` which calls `to.add_child()` (from mp_tree) which then calls its own `process()`, which finally does `obj.save()`

I'm open for suggestions

Reply:
@kaedroho this is the last bit that I can't quite fix elegantly, but @gasman gave his blessing for the PR. So... file under "won't fix" ?